File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.py
import sys
readline = sys.stdin.readline

x = int(readline())
y = int(readline())

match (x, y):
    case _ if x > 0 and y > 0:
        print(1)
    case _ if x < 0 and y > 0:
        print(2)
    case _ if x < 0 and y < 0:
        print(3)
    case _ if x > 0 and y < 0:
        print(4)

File: 백준/Bronze/14681. 사분면 고르기/사분면 고르기.swift
import Foundation

let x = Int(readLine()!)!
let y = Int(readLine()!)!

switch (x, y) {
case let (a, b) where a > 0 && b > 0:
    print(1)
case let (a, b) where a < 0 && b > 0:
    print(2)
case let (a, b) where a < 0 && b < 0:
    print(3)
case let (a, b) where a > 0 && b < 0:
    print(4)
default:
    break
}
